### PR TITLE
Get rid of more XSLT macros.

### PR DESCRIPTION
**Pull request by matejak**
* Converted the product-name-macro.

### CODE REVIEW

File: chromium/product.yml [hunk 1]
```diff
@@ -6,3 +6,5 @@ profiles_root: "./profiles"
 
 cpes:
   - "cpe:/a:google:chromium-browser"
+
+full_name: Google Chromium
```
Comment (mpreisler) [inline chromium/product.yml]: just FYI, Google Chromium doesn't exist. It's either Chromium or Google Chrome.

**Comment by mpreisler** [comment]
Looks good. I like that you are keeping full product names in yaml. That should prove much better than the constantly out of date map_product_module.py

**Comment by matejak** [comment]
That's it for the time being, further work is complicated by caching - as all macros are cached at read-time, they lose ability to reflect changes made to global variables, which makes e.g. `os-type-macro` impossible to replace in non-hackish way.

File: ubuntu1404/product.yml [hunk 2]
```diff
@@ -10,3 +10,5 @@ init_system: "systemd"
 
 cpes:
   - "cpe:/o:canonical:ubuntu_linux:14.04"
+
+full_name: Ubuntu 1404
```
Comment (mpreisler) [inline ubuntu1404/product.yml]: this should be 14.04

File: ubuntu1604/product.yml [hunk 3]
```diff
@@ -10,3 +10,5 @@ init_system: "systemd"
 
 cpes:
   - "cpe:/o:canonical:ubuntu_linux:16.04"
+
+full_name: Ubuntu 1604
```
Comment (mpreisler) [inline ubuntu1604/product.yml]: 16.04

I am also wondering if we should include the nickname for the release. I know a lot of ubuntu users know the nicknames better than release numbers.

**Comment by mpreisler** [comment]
I suggested 2 more super minor changes that I didn't notice before, then it can be merged.

**Comment by matejak** [comment]
Fixed by rebase. For the reference, those values have been exported from respective `transforms/constants.xslt`, but the full name constant is still used elsewhere, so it has to stay there.

**Pull request merged**